src/app/about/about.component.spec.ts

```diff
@@ -20,6 +20,7 @@ import { AdvancedsearchComponent } from '../advancedsearch/advancedsearch.compon
 import { SearchBarComponent } from '../search-bar/search-bar.component';
 import { ContactComponent } from '../contact/contact.component';
 import { ModalComponent } from 'ng2-bs3-modal/ng2-bs3-modal';
+import { TopbarComponent } from '../topbar/topbar.component';
```

Comment:
I don't find a reason in making the topbar component. it just increases the lines of code.
see your component file it will be empty with no functionality.
Ideally there should be some individual behaviour for separating into a component